### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [leozide/leocad#803] [Ideas]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by leozide** [comment]
It's already available as deb, appimage and ubuntu store. What are the advantages of pacstall over those?

**Reply by Elsie19** [reply]
Well, you could ship a deb *and* appimage version with Pacstall. The main advantage is the speed at which you can get applications, as the ubuntu repos aren't the fastest, and if a working PR is submitted to our repository, it's usually merged within the first 4 hours of creation, so the most recent version of leocad could be available to anyone who uses Pacstall.
